### ISSUE TITLE
Remove unique_ids line which causes problems and is not used

### ISSUE THREAD

**Issue by Nuanda** [wjurkowski/ONION#41]
I think `SimilarityClustering.R#prep` line:

```
unique_ids <- makeUnique(sdfid(sdfset))
```

is not used at the moment, right? If it is so, I'd propose to remove it since it causes problems with some SDF libraries:

```
Error in seq.default(1, dupids[i]) : 'to' cannot be NA, NaN or infinite
```

This is probably connected to the fact that these libraries do not have anything in the molecule definition's 1st header line (blank molecule id).

**Comment by wjurkowski** [comment]
Yes, maybe we could have the redundancy as part of QC health check after user library upload

**Comment by Nuanda** [comment]
I'm not sure you you need to force uniqueness in such a way. It look like the method will apply its own unique cids (CMP1, CNP2...) anyway. Or, as I do, one can reassign cid with one's own unique ID set.

Can you remove that line?
